test/imminence/imminence_api_pact_test.rb

```diff
@@ -17,7 +17,7 @@
           method: :get,
           path: "/places/number-plate-supplier.json",
           headers: GdsApi::JsonClient.default_request_headers,
-          query: { lat: "1.11", lng: "2.22", limit: "5" },
+          query: { lat: "-2.01", lng: "53.1", limit: "5" },
```

Comment:
Could this be rebased into that commit too?
And then in your commit message - or in an inline comment in the code here - explain why these coordinates are used. If I remember correctly, they're somewhat arbitrary but should be similar to those added in https://github.com/alphagov/imminence/pull/644, but not exactly the same since we want to test the proximity logic works properly.

Reply:
👍